graphql/queries/addComment.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from '@apollo/client'
+
+const ADD_COMMENT = gql`
+  mutation addComment(
+    $line: Int!
+    $submissionId: Int!
+    $content: String!
+    $fileName: String!
+    $authorId: Int!
```

Comment:
`authorId` shouldn't be a parameter, you need to get it from the session in the resolver from the user that sent the request. Otherwise someone could impersonate another person when sending a comment.

Reply:
Makes sense. I will change it.